src/core.e/PseudoStash.pm6

```diff
@@ -359,7 +359,7 @@ my class PseudoStash is Map {
                 nqp::stmts(
                     (my $ctx := nqp::decont(@ctx-info[0])),
                     nqp::if(
-                        nqp::existskey($ctx,nqp::unbox_s($key)),
+                        (nqp::isconcrete($ctx) && nqp::existskey($ctx,nqp::unbox_s($key))),
```

Comment:
I poked some more and I think the underlying problem is this: https://github.com/rakudo/rakudo/issues/1613

The ```nqp::isnull``` checks for ```$!ctx``` in ```method next-ctx()``` don't really work on the JVM backend. I played around with using ```Mu``` and changing the checks to ```nqp::eqaddr($!ctx,Mu)``` and that seemed to help -- this ```isconcrete``` check wasn't necessary anymore.

Since there are a lot of ```nqp::isnull``` checks, it doesn't seem feasible to special case for the JVM backend (with ```#?if jvm```). *sigh*